dev/provider_packages/refactor_provider_packages.py

```diff
@@ -26,7 +26,7 @@
 from fissix.fixer_util import Comma, KeywordArg, Name
 from fissix.pytree import Leaf
 
-from provider_packages.prepare_provider_packages import (
+from dev.provider_packages.prepare_provider_packages import (
```

Comment:
I know we don't like relative imports in Airflow for some reason, but :)

```suggestion
from .prepare_provider_packages import (
```

Reply:
I still do not like relative imports. They introduce more problems that they solve. I prefer to make assumption that the root of our repository is root of pythin sources and start from there. This is a really safe assumption